Dockerfile-base-ttyd-iks

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:18.04
+
```

Comment:
I think the name for this dockerfile should be different: something representing the main lang installed and istio, to follow the “convention” we’ve created thus far. Perhaps `Dockerfile-istio`? We can leave the `ttyd` out of this entirely, as this is all based on it to begin with.